fragments/itemcollection/openapi.yaml

```
@@ -31,6 +31,7 @@ components:
             $ref: '../../core/commons.yaml#/components/schemas/link'
           example:
             - rel: next
+              type: application/json
```

Minor, but I think the search results will be GeoJSON, right?

```suggestion
              type: application/geo+json
```